experimental/tf_proxy/server/src/grpc.rs

```diff
@@ -54,3 +60,42 @@ impl BackendConnection {
         PredictionServiceClient::new(self.channel.clone())
     }
 }
+
+pub async fn handle_request(
+    mut client: PredictionServiceClient<Channel>,
+    cleartext_request: Vec<u8>,
+) -> anyhow::Result<Vec<u8>> {
+    let predict_request =
+        PredictRequest::decode(&*cleartext_request).context("couldn't parse decrypted request")?;
+    // Wrap the gRPC client call to make it `Sync`.
+    let sync_predict = FutureSyncWrapper {
+        inner: SyncWrapper::new(client.predict(predict_request)),
```

Comment:
Could you please add more comments on what it tries to predict?

Reply:
Done.